components/borrow.tsx

```diff
@@ -368,9 +371,9 @@ const Borrow = ({
                 <Asset
                   key={i}
                   currency={assetStatus.underlying}
-                  apy={apy[assetStatus.underlying.address] ?? 0}
-                  available={available[assetStatus.underlying.address] ?? 0n}
-                  borrowed={borrowed[assetStatus.underlying.address] ?? 0n}
+                  apy={12.34 ?? 0}
+                  available={100000000000n ?? 0n}
```

Comment:
Will this be handled in a different PR?

Reply:
https://github.com/clober-dex/coupon.finance/pull/77/commits/c6a55b42da247d3711b55140318147ca06a3e413 @onemonster Good